src/test/java/com/pontusvision/gdpr/PVTemplateTests.java

```diff
@@ -609,9 +608,58 @@ public void test00008TemplateDataBreachReport() throws InterruptedException {
       String expectedReport = "People: 4\n" +
               "Data Sources: 2\n" +
               "Servers: 2\n";
-      assertEquals(expectedReport, report, "Expecting ROPA to have a Lawful Basis");
+      assertEquals(expectedReport, report, "Data Breache's statistics/numbers");
+
+      req.setReportTextBase64(
+              Base64.getEncoder().encodeToString(("{{ context.Event_Data_Breach_Description | " +
+                      "default('Favor Preencher o campo <b>Descri&ccedil;&atilde;o</b> na Lista " +
+                      "<b>Incidentes de Seguran&ccedil;a Reportados</b> no SharePoint') }}")
+                      .getBytes()));
+      reply = res.reportTemplateUpsert(req);
+      templateId = reply.getTemplateId();
+      contextId = App.g.V().has("Event.Data_Breach.Form_Id", P.eq("5")).id().next().toString();
+      renderReq.setRefEntryId(contextId);
+      renderReq.setTemplateId(templateId);
+      renderReply = res.reportTemplateRender(renderReq);
+      report = new String(Base64.getDecoder().decode(renderReply.getBase64Report().getBytes()));
+      expectedReport = "EXPOSIÇÃO DA OPINIÃO DOS COLABORADORES QUANTO A COMIDA DA CANTINA";
+      assertEquals(expectedReport, report, "Data Breache's Description");
 
 
+      req.setReportTextBase64(
+              Base64.getEncoder().encodeToString(("{% if context.Event_Data_Breach_Authority_Notified == 'Sim' %}" +
```

Comment:
How did this work? Shouldn't Sim be SIM??

Reply:
pois é ...

Reply:
it didn't work ... because the output was supposed to be` "O Titular dos dados j&aacute; foi notificao desse evento."` ... not `"O Titular dos dados ainda N&Atilde;O foi notificado desse evento."`